database/sql/src/main/scala/cromwell/database/slick/MetadataSlickDatabase.scala

```diff
@@ -93,6 +94,24 @@ trait MetadataSlickDatabase extends MetadataSqlDatabase {
     } yield ()
   }
 
+  private def updateCustomLabelEntry(metadataEntry: MetadataEntry)
```

Comment:
Perhaps the comment and/or naming could be more clear that this is a summary table.